core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the functionality of voiceover upload.
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var ExplorationEditorPage =
+    require('../protractor_utils/ExplorationEditorPage.js');
+
+describe('Voiceover upload features', function() {
+  var TEST_USERNAME = 'uploadUser';
+  var TEST_EMAIL = TEST_USERNAME + '@example.com';
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorTranslationTab = null;
+  var explorationEditorSettingsTab = null;
+
+  beforeAll(function() {
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+
+    users.createUser(TEST_EMAIL, TEST_USERNAME);
+    users.login(TEST_EMAIL);
+    workflow.createExploration();
+
+    explorationEditorMainTab.setStateName('Uploading translation file');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'This is the first card.'
+    ));
+    explorationEditorMainTab.setInteraction('EndExploration');
+  });
+
+  beforeEach(function() {
+    explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
+  });
+
+  it('should upload an audio file', function() {
+    explorationEditorTranslationTab.openUploadAudioModal();
+    explorationEditorTranslationTab.uploadAudio(
```

Comment:
Line 59 and 60 are like action for `should upload an audio file`, can we do something to verify this action?

(Maybe once the audio is uploaded the color changes in graph or the nav-bar of the translation content container.)

Reply:
> (Maybe once the audio is uploaded the color changes in graph or the nav-bar of the translation content container.)

You can create a separate PR for the e2e test for Graph & navbar color indicators. For this PR you can pick something simple to verify the audio is loaded successfully like you can play the uploaded audio.

Reply:
Got it! Is the test for Graph & navbar color going to be just for voiceoverUploadFeatures?

Reply:
Yeah, as colors on graph nodes changes on uploading/recording audio files. Would you like work on that?